prod/westeurope/internal/api/apim/api_management_api_io_payportal/policy.xml

```diff
@@ -8,11 +8,17 @@
     <rate-limit-by-key calls="20" renewal-period="30" counter-key="@(context.Request.IpAddress)" />
     <cors>
       <allowed-origins>
-        <origin>https://io-p-cdnendpoint-iopayportal.azureedge.net/</origin> <!-- CDN frontend io-payportal -->
-        <origin>https://io-p-cdnendpoint-iopay.azureedge.net/</origin>
-        <origin>https://io.italia.it/</origin>
-        <origin>https://www.pagopa.gov.it/</origin>
-        <origin>http://localhost:8081/</origin>
+        <choose>
```

Comment:
This policy is not valid.
When I run the apply the azure api call fails with the following error:

```
Error: creating or updating API Policy (Resource Group "io-p-rg-internal" / API Management Service "io-p-apim-api" / API "io-payportal-api"): apimanagement.APIPolicyClient#CreateOrUpdate: Failure responding to request: StatusCode=400 -- Original Error: autorest/azure: Service returned an error. Status=400 Code="ValidationError" Message="One or more fields contain incorrect values:" Details=[{"code":"ValidationError","message":"Error in element 'cors' on line 9, column 6: The element 'allowed-origins' has invalid child element 'choose'. List of possible elements expected: 'origin'.","target":"cors"}]
```

I've tried to fix it by myself but it seems you can't use conditions with cors.
I've also tried the suggestion here: https://feedback.azure.com/forums/248703-api-management/suggestions/40112266-allow-conditional-cors-policy-in-the-inbound-pol
but it fails too.

Reply:
ok, so the solution would be to create a policy at the operation level? as suggested before?

Reply:
we can try to create a policy at the operation level on azure portal to check if it is valid

Reply:
It didn't work to me .... maybe I did something wrong.

Reply:
It works with override policy rule at operation level, we need to create our api operation module to manage custom policy https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/api_management_api_operation_policy

Reply:
@pasqualedevita @infantesimone we need this module: https://pagopa.atlassian.net/browse/IOINFRA-58
to create a policy associated to the specific API.